irmark1/templates/complete.py

```diff
@@ -117,9 +125,17 @@ def drive(cfg, model_path=None, use_joystick=False, model_type=None, camera_type
             cam = RS_D435i(image_w=cfg.IMAGE_W, image_h=cfg.IMAGE_H, image_d=cfg.IMAGE_DEPTH, framerate=cfg.CAMERA_FRAMERATE)
         else:
             raise(Exception("Unkown camera type: %s" % cfg.CAMERA_TYPE))
-            
-        V.add(cam, inputs=inputs, outputs=['cam/image_array'], threaded=threaded)
-        
+
+        if cfg.CAMERA_TYPE == "D435i":
+            V.add(cam, inputs=inputs, outputs=['cam/image_array_a', 'cam/image_array_b', 'cam/imu_array'], threaded=threaded)
+            from irmark1.parts.image import StereoPair
+            V.add(StereoPair(), inputs=['cam/image_array_a', 'cam/image_array_b'],
+            outputs=['cam/image_array'])
+        else:
+            V.add(cam, inputs=inputs, outputs=['cam/image_array'], threaded=threaded)
+            from irmark1.parts.image import Duplicator
+            V.add(Duplicator(2), inputs=['cam/image_array'], outputs=['cam/image_array_a', 'cam/image_array_b'])
```

Comment:
Shouldn't use duplicator in drive.

Reply:
No we shouldn't. This part is basically designed for validating that the dual camera work. If we have confirmed that, we could remove it.